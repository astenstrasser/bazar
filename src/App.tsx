import React from "react";
import "./App.css";
import Header from "./header/Header";

const App: React.FC = () => {
  return (
    <div className="App">
      <Header />
      <div>produto 1</div>
    </div>
  );
};

export default App;
