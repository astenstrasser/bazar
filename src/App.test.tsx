import React from "react";
import { render } from "@testing-library/react";
import App from "./App";

test("renders snapshot", () => {
  const { container } = render(<App />);
  expect(container).toMatchSnapshot();
});
