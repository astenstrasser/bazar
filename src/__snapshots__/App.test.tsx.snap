// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders header 1`] = `
<div
  class="App"
>
  <div
    class="header"
  >
    <img
      alt="logo bazar"
      class="header-logo"
      src="logo.png"
    />
  </div>
  <div>
    produto 1
  </div>
</div>
`;
